charts/nfs-server-provisioner/templates/statefulset.yaml

```
@@ -117,6 +117,9 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- if (or .Values.priorityClass.name .Values.priorityClass.create)  }}
+      priorityClassName: {{ .Values.priorityClass.name | default (include "nfs-provisioner.fullname" .) | quote }}
+      {{- end }}
```

I think it's better to make it explicit,
`--set priorityClass.enabled=false --set priorityClass.name=blabla` shouldn't deploy the priorityclass resource

```suggestion
      {{- if .Values.priorityClass.create }}
      priorityClassName: {{ .Values.priorityClass.name | default (include "nfs-provisioner.fullname" .) | quote }}
      {{- end }}
```